src/core_modules/capture-core/components/TeiSearch/epics/teiSearch.epics.js

```diff
@@ -34,7 +34,7 @@ import { getSearchFormId } from '../getSearchFormId';
 import type { QuerySingleResource } from '../../../utils/api/api.types';
 
 const getOuQueryArgs = (orgUnit: ?Object, orgUnitScope: string) =>
-    (orgUnitScope !== 'ACCESSIBLE' ?
+    (orgUnitScope !== 'ACCESSIBLE' || 'CAPTURE' || 'ALL' ?
```

Comment:
Hey,
This line does not check the `orgUnitScope` value against the consts `'CAPTURE'` or `'ALL'`. Below are some suggestions to make the logic work: 
`orgUnitScope !== 'ACCESSIBLE' && orgUnitScope !== 'CAPTURE' && orgUnitScope !== 'ALL'`
or
`!['ACCESSIBLE', 'CAPTURE', 'ALL'].includes(orgUnitScope)`

Thanks!